pkg/lightning/backend/remote/chunk_sender.go

```diff
@@ -0,0 +1,393 @@
+// Copyright 2025 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package remote
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"sync/atomic"
+	"time"
+
+	"github.com/pingcap/errors"
+	"github.com/pingcap/tidb/pkg/lightning/common"
+	"github.com/pingcap/tidb/pkg/lightning/log"
+	"go.uber.org/zap"
+)
+
+// PutChunkResult is json data that returned by remote server PUT API.
+type PutChunkResult struct {
+	FlushedChunkID uint64 `json:"flushed-chunk-id"`
+	HandledChunkID uint64 `json:"handled-chunk-id"`
```

Comment:
what's the difference between flushed and handled?

Reply:
I added comment